lang/python/setup_pip.py

```diff
@@ -329,31 +261,31 @@ def source_filter(sources):
         d = os.path.join(stage_dir, os.path.dirname(f))
         if not os.path.isdir(d):
             os.makedirs(d)
-        if f in movers:
-            src = movers[f]
-        else:
-            src = f
         # Symlinks are not followed in setup, we need to use real files.
-        shutil.copy2(src, os.path.join(stage_dir, f))
+        shutil.copy2(f, os.path.join(stage_dir, f))
+    # Copy files in lang/python/wiredtiger to the root folder.
+    pywt_path = os.path.join(stage_dir, "lang", "python", "wiredtiger")
+    shutil.copytree(pywt_path, stage_dir, dirs_exist_ok=True)
+    shutil.move(os.path.join(stage_dir, 'init.py'), os.path.join(stage_dir, '__init__.py'))
     os.chdir(stage_dir)
     sys.argv.append('--dist-dir=' + os.path.join('..', 'dist'))
 else:
-    sources = [ os.path.join(python_rel_dir, 'wiredtiger_wrap.c') ]
+    sources = [ os.path.join(conf_make_dir, python_rel_dir, 'CMakeFiles', '__wiredtiger.dir', 'wiredtigerPYTHON_wrap.c') ]
 
 wt_ext = Extension('_wiredtiger',
     sources = sources,
     extra_compile_args = cflags + cppflags,
     extra_link_args = ldflags,
     libraries = builtin_libraries,
-    extra_objects = [ os.path.join(build_dir, '.libs', 'libwiredtiger.a') ],
     include_dirs = inc_paths,
     library_dirs = lib_paths,
 )
 extensions = [ wt_ext ]
 env = { "CFLAGS" : ' '.join(cflags),
         "CPPFLAGS" : ' '.join(cppflags),
         "LDFLAGS" : ' '.join(ldflags),
-        "PATH" : build_path }
+        "CMAKE_LIBRARY_PATH" : os.getenv("CMAKE_LIBRARY_PATH"),
```

Comment:
If `os.getenv`can't find the env variable it will default return `None`, which I think will fail when passing it into `call()` . You should be able override the default value though with an empty string e.g.: `os.getenv('CMAKE_LIBRARY_PATH', default='')` 
https://docs.python.org/3/library/os.html#os.getenv

Reply:
For context, run I run the script on my mac, it crashes due to a `None` value in the build environment.

Reply:
Thanks for the tip.

Reply:
Fixed and tested.